Cargo.lock

```diff
@@ -240,12 +240,13 @@ dependencies = [
  "conllx 0.11.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "failure 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
  "finalfrontier 0.6.0",
+ "finalfusion 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This time the Cargo.lock change is necessary since I need trait bounds `Into<finalfusion::VocabWrap>` on the `Vocab` type. Are you okay with upgrading the `serde` dependency at the same time?

Reply:
I still don't understand why this is happening. `Cargo.lock` should only be updated if you update a dependency in `Cargo.toml` or if you explicitly run `cargo update`.

I try to do `cargo update` in a separate commit (or sometimes in the same commit when bumping the crate version). It's nice if a bug gets introduced in a dependency, that you can bisect it to a commit that updates `Cargo.lock`. If `Cargo.lock` has updates unrelated to the rest of the commit, you have to find out first whether the code changes introduced the bug or the dependencies bump.

Of course, if a dependency is added (as it is here), it is of course necessary to commit the corresponding changes in `Cargo.toml`.

Reply:
I'm not any wiser, but I made sure that it wasn't related to IntelliJ's Rust plugin by editing the line through vim. The result was the same, the minor version of `serde` gets bumped to `92` from `89` as soon as I `cargo build --release`.

Do you want me to split the update of the dependency into a seperate PR?

Reply:
Yeah, but you could then run `cargo update` to update all the dependencies that have a newer version (within the bounds of `Cargo.toml`)